src/main/java/com/melihsurkmez/secondhand/ecommerce/model/User.java

```diff
@@ -0,0 +1,26 @@
+package com.melihsurkmez.secondhand.ecommerce.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Data
+@NoArgsConstructor
+@RequiredArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name="users")
+public class User {
```

Comment:
Did you forget to use `BaseEntity`?

Reply:
I didnt forget. This class has not been used in 'FolksDev-Spring Boot Ecommerce' yet.